instana/instrumentation/asyncio.py

```diff
@@ -14,27 +14,33 @@
     @wrapt.patch_function_wrapper('asyncio','ensure_future')
     def ensure_future_with_instana(wrapped, instance, argv, kwargs):
         if config['asyncio_task_context_propagation']['enabled'] is False:
-            return wrapped(*argv, **kwargs)
+            from opentracing.scope_managers.contextvars import no_parent_scope
```

Comment:
I think this import could be moved out of this if statement:

1. Becase by default this branch is taken anyway, 
    most users have no idea that this branch can even be avoided, and this import is executed repeatedly, [which has an import statement overhead](https://wiki.python.org/moin/PythonSpeed/PerformanceTips#Import_Statement_Overhead).
2.  The point of the import guard around this file is to find out import issues sooner, rather than later.
3.  The [Python style also prefers putting imports to the top](https://peps.python.org/pep-0008/#imports).
4.  It is less lines to read when trying to focus on the logic, and not on where certain names are coming from.

Reply:
Got it. Moved.